components/server/src/server2.rs

```diff
@@ -1592,7 +1592,7 @@ mod test {
         // Prepare some data for two tablets of the same region. So we can test whether
         // we fetch the bytes from the latest one.
         for i in 1..21 {
-            tablet.put_cf(CF_DEFAULT, b"key", b"val").unwrap();
+            tablet.put_cf(CF_DEFAULT, b"zkey", b"val").unwrap();
```

Comment:
Curious about why adding prefix `z`?

Reply:
Non data keys are filtered out. Previously using `Decision::Remove` these keys are simply replaced with deletion marks. Now with `SkipUntil` these keys are completely removed so no SST is generated and pending bytes stay unchanged. Adding the prefix makes sure they won't be filtered.